data/styles/point_dynamic_icon.ts

```diff
@@ -0,0 +1,16 @@
+import { Style } from 'geostyler-style';
+
+const pointDynamicIconPoint: Style = {
+  name: 'OL Style',
+  rules: [
+    {
+      name: 'OL Style Rule 0',
+      symbolizers: [{
+        kind: 'Icon',
+        image: '{{path}}',
```

Comment:
why double brackets? we are trying to use this with GeoServer styles and they use $ brackets - `${path}`

Reply:
The double brackets are only used within the parsing context. This is basically just a hint for the parser to not just write a static url, but to use the styling-format dependent way of providing dymanic icons. In the case of openlayers styles, the parser creates a style function instead of a style object.

So at the end, the double brackets should not be part of the parsed style

Reply:
Yes, I get the reason for the placeholders, I'm curious about the choice to use double brackets vs the `${value}` convention. 

Maybe our use case is not very common, but this might make more sense if I explain... we want to take existing SLDs (from GeoServer) and use geostyler to conver them into OpenLayers styles in order to keep a consistent style between WMS rendered map images and client side renderings via WFS.  So we convert from SLD -> geostyler style -> OL style. 

When we do this we end up with placeholders using $ bracket notation that aren't understood as placeholders in the OL conversion.

Reply:
Alright, now I understand. The main problem here is that the geostyler-sld-parser does not yet support handling dynamic icon urls. So currently, the `href` attribute of `<OnlineResource>` will always be interpreted as a static url (see [here](https://github.com/geostyler/geostyler-sld-parser/blob/master/src/SldStyleParser.ts#L601)), and thereby the parser passes the `${value}` notation to the other styles.

As soon as the geostyler-sld-parser supports this feature, it should replace the SLD specific notation (`${value}`) with the geostyler-style specific notation (`{{value}})`, so other parsers who support this feature will be able to interpret it.